docs/index.md

```diff
@@ -12,7 +12,7 @@ Deptrac can be used in a CI pipeline to make sure a pull request does not
 violate any of the architectural rules you defined. With the optional Graphviz
 formatter you can visualize your layers, rules and violations.
 
-![ModelController1](examples/ControllerServiceRepository1.png)
```

Comment:
Can we delete the referenced file here?

Reply:
The file was used on another page, so I have restored it for now